src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,35 @@
 
 public class DefaultCountingOutRhymer {
 
-    private final int[] numbers = new int[12];
+    public static final int AMOUNT_OF_NUMBERS = 12;
+    public static final int IS_FULL = 11;
+    public static final int EMPTY_TABLE = -1;
+    private final int[] numbers = new int[AMOUNT_OF_NUMBERS];
 
-    public int total = -1;
+    public int total = EMPTY_TABLE;
 
     public void countIn(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
     public boolean callCheck() {
-        return total == -1;
+        return total == EMPTY_TABLE;
     }
 
     public boolean isFull() {
-        return total == 11;
+        return total == IS_FULL;
```

Comment:
I think you should maybe change this constant to FULL, because "is" is reserved for boolean types.